.swift-version

```diff
@@ -1 +0,0 @@
-3.1
```

Comment:
Since we don't support SwiftPM in ReactiveCocoa 5.x (currently) so this is not needed now. This is for [swiftenv](https://github.com/kylef/swiftenv).